### PR TITLE
Smacklek variant

### PR DESCRIPTION
**Pull request by CausesAccidents**
New definitions from Malwarebytes.

### CODE REVIEW

**Comment by YesThatAllen** [comment]
Hey @CausesAccidents Genieo is everywhere, and samples could come from anyone removing malware from the computers being cared for. That said, the PRs here give the appearance that a commercial product is being ripping off, without one's own legwork being done. 

Please make sure your actions are in line with the TOS of the products you're using, so that we can maintain a healthy relationship with vendors.

File: Genieo.adf [hunk 1]
```diff
@@ -76,6 +76,7 @@
 		<File cruft="true">/Users/*/Library/Application Support/JakeCares</File>
 		<File cruft="true">/Users/*/Library/Application Support/Javeview</File>
 		<File cruft="true">/Users/*/Library/Application Support/Jimbrie</File>
+		<File cruft="true">/Users/*/Library/Application Support/Keka</File>
 		<File cruft="true">/Users/*/Library/Application Support/Kuklorest</File>
```
Comment (YesThatAllen) [inline Genieo.adf:81]: I've used [Keka](http://www.kekaosx.com/en/) for a very long time. Are you certain this is malware?

**Comment by CausesAccidents** [comment]
That's a fair call, Allen, I'm new at this Github thing and the last thing I want is to cause undue problems. I try to double-check these files before the program removes it and they're almost always spyware. I'll make sure to rephrase my updates appropriately.

Regarding Keka, I'm fine with removing the line. I got a hit for it in Malwarebytes and removed it just in case. In the future I'll make sure to double check online for such applications.

**Comment by CausesAccidents** [comment]
Hey Allen, is this fine to merge now?

**Comment by YesThatAllen** [comment]
Yerp.

**Comment by CausesAccidents** [comment]
Sweet, thanks Allen!

**Pull request merged**